apps/verification-portal/src/lib/components/search/SearchResults.svelte

```diff
@@ -4,14 +4,22 @@
   import Card from '$lib/components/Card.svelte';
   import { flip } from 'svelte/animate';
   import ImagePlaceholder from '$lib/components/ImagePlaceholder.svelte';
+  import type { Web2DataState, AssetFeatured } from '$lib/types';
```

Comment:
Note to self: externalize types, merge typings between assets gateway and main app.